flexmeasures/data/scripts/cli_tasks/data_add.py

```diff
@@ -0,0 +1,327 @@
+@click.option("--asset-type-name", required=True)
+@click.option("--unit", required=True, help="e.g. MW, kW/h", default="MW")
+@click.option("--capacity-in-MW", required=True, type=float)
+@click.option(
+    "--event-resolution",
+    required=True,
+    type=int,
+    help="Expected resolution of the data in minutes",
+)
+@click.option(
+    "--latitude",
+    required=True,
+    type=float,
+    help="Latitude of the asset's location",
+)
+@click.option(
+    "--longitude",
+    required=True,
+    type=float,
+    help="Longitude of the asset's location",
+)
+@click.option(
+    "--owner-id", required=True, type=int, help="Id of the user who owns this asset."
+)
+@click.option(
+    "--market-id",
+    type=int,
+    help="Id of the market used to price this asset. Defaults to a dummy TOU market.",
+)
+@click.option(
+    "--timezone",
+    default="UTC",
+    help="timezone as string, e.g. 'UTC' (default) or 'Europe/Amsterdam'.",
+)
+def new_asset(**args):
+    """
+    Create a new asset.
+    """
+    check_timezone(args["timezone"])
+    # if no market given, select dummy market
+    if args["market_id"] is None:
+        dummy_market = Market.query.filter(Market.name == "dummy-tou").one_or_none()
+        if not dummy_market:
+            print(
+                "No market ID given and also no dummy TOU market available. Maybe add structure first."
+            )
+            raise click.Abort()
+        args["market_id"] = dummy_market.id
+    check_errors(AssetSchema().validate(args))
+    args["event_resolution"] = timedelta(minutes=args["event_resolution"])
+    asset = Asset(**args)
+    app.db.session.add(asset)
+    app.db.session.commit()
+    print(
+        f"Successfully created asset with ID:{asset.id}."
+        f" You can access it at its entity address {asset.entity_address}"
+    )
+
+
+@fm_add_data.command("weather-sensor")
+@with_appcontext
+@click.option("--name", required=True)
+@click.option("--weather-sensor-type-name", required=True)
+@click.option("--unit", required=True, help="e.g. °C, m/s, kW/m²")
+@click.option(
+    "--event-resolution",
+    required=True,
+    type=int,
+    help="Expected resolution of the data in minutes",
+)
+@click.option(
+    "--latitude",
+    required=True,
+    type=float,
+    help="Latitude of the sensor's location",
+)
+@click.option(
+    "--longitude",
+    required=True,
+    type=float,
+    help="Longitude of the sensor's location",
+)
+@click.option(
+    "--timezone",
+    default="UTC",
+    help="timezone as string, e.g. 'UTC' (default) or 'Europe/Amsterdam'",
+)
+def add_weather_sensor(**args):
+    """Add a weather sensor."""
+    check_timezone(args["timezone"])
+    check_errors(WeatherSensorSchema().validate(args))
+    args["event_resolution"] = timedelta(minutes=args["event_resolution"])
+    sensor = WeatherSensor(**args)
+    app.db.session.add(sensor)
+    app.db.session.commit()
+    print(f"Successfully created sensor with ID:{sensor.id}.")
```

Comment:
Consider cherry-picking 849308ee47e8cd7b96dd9df6878be12ac05dd38b so you can print out the entity address for created weather sensors (and markets), too.

Reply:
I rather added it commented out, plus a TODO for uncommenting it when #66 has landed. Not important enough to get into git trickery with potential conflicts.